src/components/DraggableList/types.ts

```diff
@@ -0,0 +1,21 @@
+import type {DraggableChildrenFn} from 'react-beautiful-dnd';
+import type {RenderItemParams as OriginalRenderItemParams} from 'react-native-draggable-flatlist';
+
+type DraggableListData<T> = {
+    data: T[];
+};
+
+type DraggableListProps<T> = {
```

Comment:
We used to have comments for all items inside `propTypes`.

Maybe can we also do the same for types for props?

Reply:
I've added comments to the props and removed some redundant props (which were previously added when it was supposed to be a library).

Reply:
thank you for all the prop comments 🙇